### ISSUE TITLE
API bug

### ISSUE THREAD

**Issue by AndreaCorallo** [PMAlgo/pmc-0.2#18]
API interface crashing

**Comment by AndreaCorallo** [comment]
now running 
make MODE=Debug pmc-api-test
cd bin/
./pmc-api-test ../datasets/2dchess_1000p_1000v_10dim_0err.float

**Comment by vesely-david** [comment]
when run commands above i get:

error while loading shared libraries: libpmc-api.so: cannot open shared object file: No such file or directory

It seems really strange to me because the libpmc-api.so actualy is present in the bin directory

**Comment by AndreaCorallo** [comment]
Are you running the executable from the bin dir?
Any way this should fix the problem.

export LD_LIBRARY_PATH=/path_to_pmc/bin:$LD_LIBRARY_PATH

PS this is the source of the small main written for test purpose https://github.com/PMAlgo/pmc-0.2/blob/master/src/pmc-api-test.c

**Comment by vesely-david** [comment]
Yeah i was, but this fixed the problem thxs! :+1: No I can run the test through the command line.
